ffi/SDL2_0_h.lua

```diff
@@ -966,4 +966,5 @@ struct SDL_version {
   Uint8 patch;
 };
 void SDL_GetVersion(struct SDL_version *) __attribute__((visibility("default")));
+void SDL_free(void *);
```

Comment:
Please also add this to https://github.com/koreader/koreader-base/blob/96b2a16e685a75de1b7fcfe30b9dd19d54f5f306/ffi-cdecl/SDL2_0_decl.c

Also I'm inclined to think stdinc should go at the very top. What do you think @NiLuJe?

Reply:
Are those `…_decl.c` files really useful? I looked at it briefly and it seems to rely on plugins for an old GCC version (4.x), so I've just been copy pasting the relevant declarations when needed.

Reply:
Yes, otherwise your lines are likely to be accidentally lost. See https://github.com/koreader/ffi-cdecl

Reply:
That is, the idea is that the _h files are auto-generated end results. We edit them manually for development because that's easier, and conversely maintenance is easier when automated.

Reply:
OK, done.